### PR TITLE
Bug 1755869 - require auth before create interactive

### PR DESCRIPTION
**Pull request by jmaher**

### CODE REVIEW

**Comment by Archaeopteryx** [comment]
With the patch applied, the interactive task [fails](https://treeherder.mozilla.org/logviewer?job_id=369469572&repo=autoland). Is mochitest-devtools on Linux x64 debug not supported? 'Similar Jobs' shows failing and passing tasks from other people.

File: ui/models/taskcluster.js [hunk 1]
```diff
@@ -15,6 +15,12 @@ export default class TaskclusterModel {
     );
   }
 
+  static async getCredentials({ currentRepo, testMode = false }) {
```
Comment (Archaeopteryx) [inline ui/models/taskcluster.js:18]: There is a [helper function](https://github.com/mozilla/treeherder/blob/cfb765d7b222b8e6a09e08187c0e98c4fb933135/ui/helpers/taskcluster.js#L71) to get the credentials - can you use that one?

File: ui/job-view/details/summary/ActionBar.jsx [hunk 2]
```diff
@@ -308,6 +308,10 @@ class ActionBar extends React.PureComponent {
     try {
       const interactiveTask = getAction(results.actions, 'create-interactive');
 
+      if (user.email === '') {
+        await TaskclusterModel.getCredentials(currentRepo);
```
Comment (Archaeopteryx) [inline ui/job-view/details/summary/ActionBar.jsx:312]: Failure: `Taskcluster: can't access property "tc_root_url", currentRepo is undefined`

**Review by Archaeopteryx — CHANGES_REQUESTED** [review]

File: ui/models/taskcluster.js (hunk 1 above)
Comment (jmaher) [reply ui/models/taskcluster.js]: I need user data, not tc creds, so I am not going this route

File: ui/job-view/details/summary/ActionBar.jsx (hunk 2 above)
Comment (jmaher) [reply ui/job-view/details/summary/ActionBar.jsx]: same story, user creds, not tc creds

**Comment by jmaher** [comment]
I updated this and it has a new thing- just require auth and don't move forward.  This seems to work for me locally if I am signed in or not signed in.

**Comment by jmaher** [comment]
@Archaeopteryx could you take a look at this again?

**Review by Archaeopteryx — APPROVED** [review]

**Pull request merged**